src/tests/CMakeLists.txt

```diff
@@ -14,6 +14,16 @@ include_directories("${CLR_SRC_NATIVE_DIR}")
 # Add this subdir. We install the headers for the jit.
 add_subdirectory(../coreclr/pal/prebuilt/inc ../coreclr/pal/prebuilt/inc)
 
+
+# tests in the Debug configuration depend on the coreclrpal static library if they use assert
+if (NOT CLR_CMAKE_HOST_WIN32)
+  # Add component names that the pal build expects
+  add_component(${CMAKE_INSTALL_DEFAULT_COMPONENT_NAME})
+  add_definitions(-DUSE_STL)
+  include_directories(../coreclr/pal/inc/rt ../coreclr/pal/inc ../coreclr/inc)
+  add_subdirectory(../coreclr/pal/src ../coreclr/pal/src)
+endif()
```

Comment:
I don't know if this is the proper way to get the coreclrpal target to build.  It works but I'm not sure if i'm doing something wrong here.

Reply:
I am a bit confused. If the profiler test needs anything from PAL, then it would not build before. Let me try to understand what's going on.

Reply:
Ah, I see, it is using some headers that pull in the PAL headers and with the change in #64631, the _ASSERTE references the two PAL functions. I don't think we should let the profiler test pull in the PAL just because of that. I would replace the usage of _ASSERTE by assert in the test instead.